src/c++/perf_analyzer/constants.h

```diff
@@ -27,16 +27,18 @@
 
 #include <cstdint>
 
+// namespace pa = triton::perfanalyzer;
```

Comment:
Can be removed.

Reply:
Missed this. I will fix this in the next PR.